packages/assets/urls.js

```diff
@@ -644,3 +644,235 @@ export function getAssetUrlsByMetaUrl(opts) {
 		},
 	}
 }
+
+export function getAssetUrls(opts) {
```

Comment:
what if we write this into a `local.js`? I think this file might cause errors if you import it from a bundler that doesn't support the special meta url syntax. So you would do `import {getAssetUrls} from '@tldraw/assets/local'` or something??? maybe local isn't the best name though i'm not sure

Reply:
Used `selfHosted` name, but also unsure about it. Decided against local as this option is also quite useful for hosting via a CDN (you just pass in CDN's base url), so local didn't feel right for that.